### PR TITLE
update installation.txt

### PR DESCRIPTION
**Pull request by KarelWintersky**
with gentoo recipe

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -333,4 +352,4 @@ To run container with Ubuntu add ubuntu-miror to your vagga settings file
 In your vagga.yaml select proper architecture::
 
        setup:
-       - !UbuntuRelease { codename: xenial, arch: armhf }
```
Comment (tailhook) [inline docs/installation.rst]: What is the change here? Whitespace?

File: docs/installation.rst [hunk 2]
```diff
@@ -269,6 +269,25 @@ Also make sure you have musl-gcc in your path::
...
+   
+Describe vagga version
+
+   $ git describe 
+   
+For example: `v0.8.1-19-g372bded`
+
+Compile vagga with dockerized rust:
+
+   $ docker run --rm --user "$(id -u)":"$(id -g)" -e VAGGA_VERSION=v0.8.1-19-g372bded -v "$PWD":/usr/src/myapp -w /usr/src/myapp rust cargo build --release
```
Comment (tailhook) [inline docs/installation.rst]: If you're already using `$(id -u)` you might want to use `-e VAGGA_VERSION=$(git describe)`, right?

File: docs/installation.rst [hunk 3]
```diff
@@ -269,6 +269,25 @@ Also make sure you have musl-gcc in your path::
 Then just build using cargo and the appropriate target::
 
     $ cargo build --target x86_64-unknown-linux-musl
+    
+Build from source (Gentoo, docker)
```
Comment (tailhook) [inline docs/installation.rst]: What's gentoo-specific here? Maybe just "Building From Source Using Docker"?

File: docs/installation.rst [hunk 4]
```diff
@@ -269,6 +269,25 @@ Also make sure you have musl-gcc in your path::
...
+==========================
+
+Clone vagga repository
```
Comment (tailhook) [inline docs/installation.rst]: I believe all these needs double-colon for proper formatting

File: docs/installation.rst (hunk 2 above)
Comment (KarelWintersky) [reply docs/installation.rst]: in my environment `-e VAGGA_VERSION=$(git describe)` does not work :(

File: docs/installation.rst (hunk 1 above)
Comment (KarelWintersky) [reply docs/installation.rst]: yeah, i think so.

File: docs/installation.rst (hunk 4 above)
Comment (KarelWintersky) [reply docs/installation.rst]: fixed

File: docs/installation.rst (hunk 2 above)
Comment (KarelWintersky) [reply docs/installation.rst]: I don't know, why it does not work, but without `-e VAGGA_VERSION=v0.8.1-19-g372bded` it drops with complilation error (due VAGGA_VERSION will be empty)

File: docs/installation.rst (hunk 3 above)
Comment (KarelWintersky) [reply docs/installation.rst]: fixed

**Pull request merged**